jerry-ext/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required (VERSION 2.8.12)
+set(JERRY_EXT_NAME jerry-ext)
+project (${JERRY_EXT_NAME} C)
+
+# Include directories
+set(INCLUDE_EXT "${CMAKE_CURRENT_SOURCE_DIR}/include")
+
+# Source directories
+file(GLOB SOURCE_EXT args/*.c)
+
+add_library(${JERRY_EXT_NAME} STATIC ${SOURCE_EXT})
+
+target_include_directories(${JERRY_EXT_NAME} SYSTEM PUBLIC ${INCLUDE_EXT})
```

Comment:
Sorry that I am not familiar with cmake. I copy-paste the other cmakelist files in the project. 
I will remove the `SYSTEM`